src/components/header.js

```diff
@@ -13,4 +13,4 @@ const Header = () => (
     </div>
   </div>
 );
-export default Header;
\ No newline at end of file
+export default Header;
```

Comment:
Added a newline in accordance with the [eol-last](https://eslint.org/docs/rules/eol-last) rule.